Orm/Xtensive.Orm/Orm/Linq/Translator.Queryable.cs

```diff
@@ -27,6 +27,8 @@ namespace Xtensive.Orm.Linq
   internal sealed partial class Translator : QueryableVisitor
   {
     private static readonly Type IEnumerableOfKeyType = typeof(IEnumerable<Key>);
+    private static readonly ParameterExpression tupleParameter = Expression.Parameter(WellKnownOrmTypes.Tuple, "tuple");
+    private static readonly ParameterExpression parameterContextContextParameter = Expression.Parameter(WellKnownOrmTypes.ParameterContext, "context");
```

Comment:
Naming policy violation. Correct name example is one line above.

Reply:
done